.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: c
+
+dist: trusty
+sudo: required
```

Comment:
why do we need sudo ?  the build/tests shouldn't need this.  i guess it's because you want to use apt below ?

it'd be better if we could just rely on addons->apt->packages to list the packages we care about.  if that means using an older musl package, or dropping that coverage entirely, then i think that's OK.

Reply:
On the other hand, what's the harm using sudo here?

Reply:
it adds overhead to the system that isn't needed otherwise, you're pulling from a repo that is effectively random from our perspective, and why do we care about whether musl is the latest ?  we don't do that for gcc, clang, glibc, or other packages.  arguably, we should be building against the versions other people have available.